pvlib/irradiance.py

```diff
@@ -2242,6 +2297,10 @@ def liujordan(zenith, transmittance, airmass, dni_extra=1367.0):
     return irrads
 
 
+liujordan = deprecated('0.8', alternative='campbellnormam',
+                       name='liujordan', removal='0.9')(liujordan)
```

Comment:
I don't think there's anything wrong with this, but it might be a little more clear if we change the function definition to `_liujordan` so that readers of the code cannot miss the deprecation wrapper.

Reply:
Not needed if the deprecation warning and `@fail_on_pvlib_version('0.9')` are made part of the old `test_liujordan`, right?

Reply:
remove `irradiance.liujordan` from api.rst?

Reply:
yes